### ISSUE TITLE
[HW2] provideYearData 관련 질문입니다

### ISSUE THREAD

**Issue by jiyewise** [swsnu/swppfall2020#36]
provideYearData 함수를 구현하는 중인데, 혹시 각 해마다 남자의 수와 여자의 수가 같다고 생각해도 되나요? (e.g. 남자의 rank가 1-500까지 있으면 여자의 rank도 1-500까지 있는지)
그리고 현재 기준이 되는 babyname.report.csv에는 남녀 각 rank가 1000까지 있는데, 이 rank의 수는 1000으로 고정되지 않고 바뀔 수 있나요?
감사합니다.

**Comment by hy00nc** [comment]
네, HW1에서 작성한 로직으로 생성된 output이므로, 남자 여자 둘 다 각각 상위 1000개 이름을 가져온 것이기 때문에 남자 여자 이름 수가 매년 동일하고, rank 수도 1000개까지 있는게 맞습니다. 

추가로 말씀드리자면 위와 같이 데이터 수를 1000개로 고정해서 코딩하여도 오답은 아니지만, 어떤 데이터의 집합이 주어졌을 때 데이터 수와 관계 없이 모든 데이터에 대해서 함수를 적용 하는 것이 이번 과제의 핵심이라고 할 수 있습니다.

**Comment by jiyewise** [comment]
답변 감사드립니다!

**Comment by hy00nc** [comment]
정정하겠습니다 죄송합니다. 데이터 갯수를 가정하지 말고 작성해주시기 바랍니다.

**Comment by jiyewise** [comment]
네 감사합니다!
